Source/WebCore/Modules/url-pattern/URLPatternTokenizer.cpp

```diff
@@ -43,7 +43,12 @@ static bool isValidNameCodepoint(UChar codepoint, bool first)
 
 bool Token::isNull() const
 {
-    return type == TokenType::Open && !index && !value;
+    return !index;
+}
+
+bool Token::isInitialized() const
```

Comment:
For me, I think it was a lot more readable to make a function isInitialized rather than doing !isNull(). 

But I can change it back!

Reply:
Got it; yeah I think we probably want to stick to `!isNull()` if spec says "if token is not null", since that will be easier for us to reason about changes the when we compare our implementation against spec.